code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -745,8 +745,16 @@ GLOBAL_LIST_EMPTY(PDAs)
 	tnote += "<i><b>&larr; From <a href='byond://?src=[REF(src)];choice=Message;target=[REF(signal.source)]'>[signal.data["name"]]</a> ([signal.data["job"]]):</b></i><br>[signal.format_message()]<br>"
 
 	if (!silent)
-		playsound(src, 'sound/machines/twobeep_high.ogg', 50, 1)
-		audible_message("[icon2html(src, hearers(src))] *[ttone]*", null, 3)
+		if(honkamt > 9)//For clown virus.
+			var/adminname = pick(strings("admin_nicknames.json", "names", "config"))
+			honkamt -= 9
+			playsound(src, 'sound/effects/adminhelp.ogg', 50, 1)
+			audible_message("<font color='red' size='4'><b>-- Administrator private message --</b></font>", null, 3)
+			audible_message("<span class='danger'>Admin PM from-</span><span class='ooc'>[adminname]</span><span class='danger'>: [pick("What do you think you're doing?","Hey, got a minute?", "Care to explain just how you aren't powergaming right now?","We need to talk.","Stop that or I'll ban you.")]</span>", null, 3)
```

Comment:
```suggestion
			audible_message("<span class='danger'>Admin PM from-</span><span class='ooc'>[adminname]</span><span class='danger'>: [pick("What do you think you're doing?","Hey, got a minute?", "Care to explain just how you aren't powergaming right now?","We need to talk.","Stop that or I'll ban you.","What the fuck are you doing?")]</span>", null, 3)
```

Reply:
redundant

Reply:
https://file.house/1eC3.png

make it this